helm/templates/kafka-topic-creation-hook.yaml

```diff
@@ -37,29 +37,21 @@ spec:
             {{- range $topicName, $topicConfigs := $.Values.kafka.topics }}
               /opt/kafka/bin/kafka-topics.sh --create \
               --if-not-exists \
-              {{- if $.Values.kafka.address }}
               --bootstrap-server {{ $.Values.kafka.address | quote }} \
-              {{- else }}
-              --zookeeper {{ $.Values.zookeeper.address | quote }} \
-              {{- end }}
               --topic {{ $topicName | quote }} \
               {{- range $configKey, $configValue := $topicConfigs.configs }}
               {{- if or (kindIs "float64" $configValue) (kindIs "int" $configValue) (kindIs "int64" $configValue) }}
               --config {{ $configKey }}={{ int $configValue }} \
               {{- else }}
-              --config {{ $configKey }}={{ $configValue | quote }} \
+              --config {{ $configKey }}={{ $configValue | trimPrefix "[" | trimSuffix "]" | trim | quote }} \
```

Comment:
for backward compatibility for config having multiple values like `cleanup.policy="[compact,delete]"`. In latest version of kafka, valid format is `cleanup.policy="compact,delete"`